mm2src/derives/enum_from/src/from_trait.rs

```diff
@@ -81,7 +89,7 @@ impl<'a> TryFrom<&'a NestedMeta> for TraitIdentMethod<'a> {
 }
 
 /// Get the meta information about the given `attr`.
-fn get_attr_meta(attr: &Attribute, attr_ident: MacroAttr) -> Vec<NestedMeta> {
+pub(crate) fn get_attr_meta(attr: &Attribute, attr_ident: MacroAttr) -> Vec<NestedMeta> {
```

Comment:
Consider moving this function to the root `lib.rs` file as it's common for `from_trait` and `from_stringify` submodules.